draft-ietf-tls-tls13.md

```diff
@@ -2244,16 +2244,18 @@ SignatureSchemeList value:
            ed25519(0x0807),
            ed448(0x0808),
 
+           /* Legacy algorithms */
+           ecdsa_sha1(0x0203),
```

Comment:
Did you want to move rsa_pkcs1_sha1 here?  Doesn't it fall under the same basic rules?  I mean, some of us have cut one of the two out, but the same logic applies to both.  You could rationalize that by merging the description of the two into the last bullet point.